packages/app-settings/src/Developer.tsx

```diff
@@ -64,16 +65,24 @@ class Developer extends React.PureComponent<Props, State> {
         </div>
         <div className='ui--row'>
           <div className='full'>
-            <Labelled label={t('Manually enter your custom type definitions as valid JSON')}>
+            <Labelled
+              help={t('Please create a key-value pair for each of your custom structures. The key should be the name of your custom structure and the value an object containing your custom type definitions.')}
+              label={t('Manually enter your custom type definitions as valid JSON')}
+            >
               <Editor
                 className='editor'
                 code={code}
                 isValid={isJsonValid}
                 onEdit={this.onEditTypes}
               />
-            </Labelled>>
+            </Labelled>
           </div>
         </div>
+        <div className='ui--row info'>
```

Comment:
This should really be part of a `HelpOverlay`, i.e. like done in https://github.com/polkadot-js/apps/pull/1030/files#diff-1149998f281026a5e26b8bc8a0e0e64fR1

(For now would just remove this info row - but should be added to fully close the original logged issue, but shouldn't stand in the way of just the basic help added above)

Reply:
Agreed, but we just don't really have the help content yet so I thought a link to the API docs would be better than nothing.  But I'm also fine with removing it.